### PR TITLE
1470 interview notes on recruitmentpositionoverviewpage

### PR DESCRIPTION
**Pull request by nemisis84**
Link: http://localhost:3000/control-panel/recruitment/view-applicant/901e8909-401b-4bca-8465-4bb2b5e0bfcb/
Most of the frontend for the linked issue is solved. 

- Refactored page to use useQuery and useMutation
- Used react-hook-form for the notes. Saves using onBlur. The put request is not implemented and will be an own issue.

### CODE REVIEW

File: frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx [hunk 1]
```diff
@@ -114,59 +101,81 @@ export function RecruitmentApplicantAdminPage() {
...
-                  ),
-                },
-                element.recruiter_priority ? element.recruiter_priority : t(KEY.common_not_set),
-                element.interview_time ? element.interview_time : t(KEY.common_not_set),
-              ],
-            };
-          })}
+          data={
+            otherRecruitmentApplication
+              ? otherRecruitmentApplication.map((element) => {
```
Comment (Snorre98) [inline frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx:106]:
```suggestion
              ? otherRecruitmentApplications.map((element) => {
```

File: frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx [hunk 2]
```diff
@@ -114,59 +101,81 @@ export function RecruitmentApplicantAdminPage() {
...
+            onClick={() => {
+              if (recruitmentApplication?.id) {
+                adminWithdraw.mutate(recruitmentApplication.id);
+              }
+            }}
+          >
+            {t(KEY.recruitment_withdraw_application)}
+          </Button>
+        )}
+      </div>
```
Comment (Snorre98) [inline frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx:178]: Burde flyttes mellom søkands stilling tittelen og søknadsteksten.

**Review by Snorre98 — APPROVED** [review]
Se på kommentarene

File: frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx (hunk 2 above)
Comment (nemisis84) [reply frontend/src/PagesAdmin/RecruitmentApplicantAdminPage/RecruitmentApplicantAdminPage.tsx]: Done

**Comment by nemisis84** [comment]
applicationID vil ikke la seg skrive som applicationId uten at siden crasher. Vet ikke hvorfor. Endret derfor tilbake til applicationID.

**Pull request merged**